components/lib/datatable/DataTable.js

```diff
@@ -557,7 +557,7 @@ export const DataTable = React.forwardRef((inProps, ref) => {
                     updateTableWidth(frozenBodyRef.current);
 
                     if (wrapperRef.current) {
-                        updateTableWidth(DomHandler.findSingle(wrapperRef.current, '.p-virtualscroller-content'));
+                        updateTableWidth(DomHandler.findSingle(wrapperRef.current, '[data-pc-name="virtualscroller"] > table > tbody'));
```

Comment:
`[data-pc-name="virtualscroller"]  [data-pc-section="tbody"]` or add new data attr?

Reply:
I think this is OK its saying find the first table and first tbody under `[data-pc-name="virtualscroller"]`

Reply:
ok